### PR TITLE
answers for lesson 1

### PR DESCRIPTION
**Pull request by ameliasheppy**
Hi! Submitting this for review

### CODE REVIEW

**Review by jdevries3133 — COMMENTED** [review]
There's a bug in github preventing me from putting inline comments so it'll need to be one giantic one instead!

> //gives us a freaky amount on the user 
> //this method will return the system uptime in seconds 
> //creepily stalks the user for how long they've been on

Just the tip of the iceberg for how creepy software can be lol!

> //imported modules from pre made libraries don't need to the .. and /.

You'll see later in the course a `node_modules` folder appear after you `npm install` anything. There's no magic here; node just automatically looks into these files that it creates on your behalf! That's why you don't need the relative path in that case.

File: 01-node-tutorial/content/asyncfiles.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+//let's do it to write now!
+readFile('./first.txt', (err, result)=>{
+    if(err){
+        console.log(err)
+        return
+    }
+    const first = result;
+    readFile('./second.txt', 'utf8', (err, result) =>{
+        if(err){
```
Comment (jdevries3133) [inline 01-node-tutorial/content/asyncfiles.js:33]: > //callback functions like button clicks run when we're ready. this will do the same

That's good analogy!

Comment (jdevries3133) [reply 01-node-tutorial/content/asyncfiles.js:33]: Ah the bug is fixed thank the Lord

File: 01-node-tutorial/content/repl.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+//modules are encapsulated code. 
+//we only share the minimum that we want to.
+
+
+//node uses common JS under the hood and every file be default is a module. 
+
+//to export a module, set it equal to an export. Or we can export in an obj.
+
+
+//way smarter people made really coolmdules that we can use
```
Comment (jdevries3133) [inline 01-node-tutorial/content/repl.js:69]: *more experienced, not smarter

**Review by jdevries3133 — APPROVED** [review]
Awesome job! I really appreciate the comments you left; makes the process much more meaningful as a reviewer. Please keep doing that throughout the course!!

**Comment by jrmcgarvey** [comment]
Directed to wrong repository.

**Pull request closed without merging**